app/controllers/action_view/storybook/stories_controller.rb

```diff
@@ -6,7 +6,6 @@ module ActionView
   module Storybook
     class StoriesController < Rails::ApplicationController
       prepend_view_path File.expand_path("../../../views", __dir__)
-      prepend_view_path Rails.root.join("app/views") if defined?(Rails.root)
```

Comment:
I have a sense that adding the conditional here (instead of a `config.to_prepare` block) might be too early in the initialization cycle. I wonder if that assumption is off base.